exercises/farewell-workflow/README.md

```diff
@@ -0,0 +1,59 @@
+# Temporal 101 Farewell Exercise
+
+# Exercise 3: Farewell Workflow
+During this exercise, you will create an Activity function,
+register it with the Worker, and modify a Workflow to execute it.
+
+Before continuing, press Ctrl-C to stop any Worker instances still running from any
+previous exercises.
+
+As with other exercises, you should make your changes in the `practice` 
+directory. Look for "TODO" comments, which will provide hints about what
+you'll need to change. If you get stuck and need additional hints, or 
+if you want to check your work, look at the completed example in the
+`solution` directory. 
+
+## Part A: Write an Activity Function
+The `TranslateActivities.cs` file contains a method(`GetSpanishGreeting`) that uses a 
+microservice to get a customized greeting in Spanish. Follow these steps to create 
+a new Activity that gets a custom farewell message from the microservice:
+
+1. Open the `TranslateActivities.cs` file. This is located in the `practice/Workflow`
+   subdirectory under the `farewell-workflow` directory in `exercises`.
+2. Create a new Activity that will get a custom farewell message.
+   1. Copy the `GetSpanishGreeting` method to create a new Activity.
+   2. Rename the new `GetSpanishGreeting` function to `GetSpanishFarewell`.
+   3. Save your changes to this file.
+
+## Part B: Register the Activity Function
+1. Open the `Program.cs` file in the Worker subdirectory in your code editor.
+2. Add a new line that registers your new Activity with the Worker (hint: you'll use the name you used for the new method in an earlier step).
+3. Save your changes to this file.
+
+## Part C: Modify the Workflow to Execute Your New Activity
+1. Open the `GreetingWorkflow.cs` file (located in the `practice/Workflow` subdirectory) in the editor.
+2. Look for the TODO message, uncomment the lines below it, and then update this code to execute the Activity function you created.
+3. Save your changes to this file.
+
+## Part D: Start the Microservice and Run the Workflow
+First, make sure the Temporal Service is running in the background. This is done by opening a new terminal window and running the following command: `temporal server start-dev --ui-port 8080 --db-filename clusterdata.db`. For more details on this command, please refer to the `Setting up a Local Development Environment` chapter in the course. 
+
+Note: If you're using the Gitpod environment to run this exercise, you can skip this step.
```

Comment:
I would move this to the top so GitPod learners don't accidentally try to do this then realize they didn't need to. The majority of our learners use GitPod